services/frontend-service/src/ui/ActionsCart/CheckoutDialog.tsx

```diff
@@ -78,6 +88,52 @@ const ApplyButton = (props: { openNotification: (msg: string) => void; closeDial
     }
 };
 
+const SyncWindowsWarning: VFC<{
+    actions: BatchAction[];
+}> = ({ actions }) => {
+    const getOverview = useCallback((api: Api) => api.overviewService().GetOverview({}), []);
+    const overview = useUnary(getOverview);
+    switch (overview.state) {
+        case 'pending':
+            // FIXME: This spinner is invisible.  Why?
```

Comment:
This is "fixed" by removing the whole overview-fetching in `CheckoutDialog` in https://github.com/freiheit-com/kuberpult/pull/180/commits/aba3ff360b454a592078cbfb39e4bdab1e5511c6. It was not necessary, because `GetOverviewResponse` was already available elsewhere in the application and just had to be piped through to here.